client/landing/stepper/declarative-flow/internals/steps-repository/sensei-launch/index.tsx

```diff
@@ -64,6 +65,12 @@ const SenseiLaunch: Step = ( { navigation: { submit } } ) => {
 				await setAdminInterfaceStyle( siteId, 'wp-admin' );
 				return true;
 			},
+			async function refreshThemeOnAtomic() {
```

Comment:
Thanks @renatho ! 

> Just to confirm, have you tried this switch at the moment we set the theme? To not have this theme change in 2 different places.

yap I did. But the catch is it only works when we do the switch 'after' the jetpack transfer. So it had to be here.

> But considering you and Donna were able to, and it fixes it for you, I'm approving the PR.

Yes, it looks like it's an intermittent issue. I wouldn't merge it if it happened only for me. But as Donna and I both faced the same issue, there are chances of other users facing the same. So I'll go ahead and merge it.

> Should we open an issue to not forget it?

I'd have done it already. But looking at how truly intermittent it is, I'm a bit hesitant. Not sure if it's worth the deep investigation it needs.

> I'm also proposing to redirect from the onboarding to the Sensei Home https://github.com/Automattic/wp-calypso/issues/85805 (for discussion). Maybe if we go with that and we decide to add a task for it, it could be a task even in the plugin, keeping the same behavior on Calypso and self-hosted sites.

That's a good option. But I think there are a few very important things in the wp home, like verifying the email and publishing the site etc. So we should weigh well about pros and cons.